packages/pieces/approval/src/lib/actions/approval.ts

```diff
@@ -0,0 +1,19 @@
+import { createAction } from "@activepieces/pieces-framework";
+import { ExecutionType, PauseType } from "@activepieces/shared";
+
+export const approvalAction = createAction({
+  name: 'approval',
+  displayName: 'Approval',
+  description: 'Pauses the flow and waits for approval from the user',
+  props: {
+  },
+  async run(ctx) {
+    if (ctx.executionType === ExecutionType.BEGIN) {
+      ctx.run.pause({
+        pauseMetadata: {
```

Comment:
We should modify this to have `ctx.run.id` and the output should contain the urls.

also to make sure it's extensible, can we make it take an array of action?

https://github.com/activepieces/activepieces/issues/1078#issuecomment-1518716653

Reply:
done in [738358e](https://github.com/activepieces/activepieces/pull/1635/commits/738358edc436fe5455423e04a7d3cf12ec413a3a) and [eeee5b2](https://github.com/activepieces/activepieces/pull/1635/commits/eeee5b2d7ae02c796d38b62b3e48bbed19d32cca)